app/views/support/WeeklyPromotion.scala

```diff
@@ -167,20 +171,25 @@ object WeeklyPromotion {
 object PlanPicker {
 
   import model.GuardianWeeklyZones._
-  val showUpdatedPrices = false
 
-  def isInRestOfWorldOrZoneC(country: Country, showUpdatedPrices: Boolean = showUpdatedPrices): Boolean = {
-    if(showUpdatedPrices) GuardianWeeklyZones.restOfWorldZoneCountries.contains(country)
+  def showUpdatedPrices = {
+    val now = DateTime.now().withZone(DateTimeZone.UTC)
+    val threshold = DateTime.parse("2018-10-10T08:00:00").withZone(DateTimeZone.UTC)
```

Comment:
Is this time right? I thought it was 10.30 now?

Reply:
Ah, last I heard it was 09:00. I'll double-check, and I suspect it'll change more than once anyway :)

Reply:
Currently it's scheduled for: Wednesday 10th October, 10:45 BST

It changed at least twice whilst you were on holiday so don't be too surprised if it changes again 🙃